js/index.js

```diff
@@ -9,7 +9,10 @@ console.clear();
 */
 
 // --v-- write your code here --v--
-
+let bookTitle = 'Clean Code';
```

Comment:
bookTitle and bookAuthor could've been const instead of let, considering they are not changing.